cli/crates/cli/src/cli_input/dev.rs

```diff
@@ -35,7 +35,7 @@ impl DevCommand {
         let default_log_levels = if self.verbose {
             LogLevelFilters {
                 functions: LogLevelFilter::Debug,
-                graphql_operations: LogLevelFilter::Info,
+                graphql_operations: LogLevelFilter::Debug,
```

Comment:
Maybe `-vv`/`--verbose --verbose` like some CLIs do? 🙂

Reply:
Well @fbjork wanted it added to `-v`, we can discuss verbose tiers separately if we want to